telegramBot/proto/user/proto/user.proto

```
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package user;
```

Buf (1.57.2)

Files with package "user" must be within a directory "user" relative to root but were in directory "telegramBot/proto/user/proto".

(PACKAGE_DIRECTORY_MATCH)